docs/shopify_app/sessions.md

```diff
@@ -33,10 +37,24 @@ rails generate shopify_app:user_model
 ```
 This will generate a shop model and user model, which will be the storage for the tokens necessary for authentication.
 
+This will generate a user and shop model which will be the storage for the tokens necessary for authentication. To enable session persistance, you'll need to configure your `/initializers/shopify_app.rb` accordingly:
+
+```ruby
+config.shop_session_repository = 'Shop'
+config.user_session_repository = 'User'
+```
+
 The current Shopify user will be stored in the rails session at `session[:shopify_user]`
 
 Read more about Online vs. Offline access [here](https://shopify.dev/apps/auth/oauth/access-modes).
 
+### Customized Session Storage - ShopifyApp::SessionRepository
+
+`ShopifyApp::SessionRepository` allows you as a developer to define how your sessions are stored and retrieved for shops. The `SessionRepository` is configured in the `config/initializers/shopify_app.rb` file and can be set to any object that implements `self.store(auth_session, *args)` which stores the session and returns a unique identifier and `self.retrieve(id)` which returns a `ShopifyAPI::Session` for the passed id. These methods are already implemented as part of the `ShopifyApp::SessionStorage` concern but can be overridden for custom implementation.
+
+### Loading Sessions
+By using the appropriate controller concern, sessions are loaded for you. `RequireKnownShop` controller concern will load a shop session with the `installed_shop_session` helper. `EnsureHasSession` controller concern will load a user session via `current_shopify_session`. Note -- these controller concerns cannot both be included in the same controller.
```

Comment:
Do we go into a bit more detail on what each of those methods mean in practice? Might be a good place for it!